### PR TITLE
Added TestExec to allow for `loom test` to work.

### PR DESCRIPTION
**Pull request by SmilyOrg**

### CODE REVIEW

File: sdk/src/testexec/TestExecutor.ls [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    import unittest.TestResult;
+    import unittest.TestRunner;
+    
+    public class TestExecutor extends Application
+    {
+        [UnitTestHideCall]
+        override public function run():void
+        {
+            
+            //IO.write("count: "+CommandLine.getArgCount()+"\n");
```
Comment (bengarney) [inline sdk/src/testexec/TestExecutor.ls]: Do we need these comments for some reason?

File: sdk/src/test/tests/ByteArrayTest.ls [hunk 2]
```diff
@@ -408,6 +408,38 @@ package tests {
...
+            // 156 - default compression
+            // 218 - best compression
+            checkBytes(ba, [120, 156, 147, 102, 96, 96, 72, 132, 131, 164, 100, 67, 35, 99, 100, 4, 0, 124, 189, 7, 153]);
+        }
+        
+        [Test]
+        function uncompress() {
+            ba.clear();
+            // Default compression header
+            fillBytes(ba, [120, 156, 147, 102, 96, 96, 72, 132, 131, 164, 100, 67, 35, 99, 100, 4, 0, 124, 189, 7, 153]);
```
Comment (bengarney) [inline sdk/src/test/tests/ByteArrayTest.ls]: Nice tests.

**Comment by bengarney** [comment]
One nit to tackle please!

**Comment by SmilyOrg** [comment]
Tackled.

**Comment by bengarney** [comment]
Thank you!

**Pull request merged**